pkg/lang/ir/v1/compile.go

```diff
@@ -256,20 +253,8 @@ func (g generalGraph) EnvString() []string {
 }
 
 func (g generalGraph) DefaultCacheImporter() (*string, error) {
-	// The base remote cache should work for all languages.
-	var res string
-	if g.CUDA != nil {
-		res = fmt.Sprintf(
-			"type=registry,ref=docker.io/%s/python-cache:envd-%s-cuda-%s-cudnn-%s",
-			viper.GetString(flag.FlagDockerOrganization),
-			version.GetVersionForImageTag(), *g.CUDA, g.CUDNN)
-	} else {
-		res = fmt.Sprintf(
-			"type=registry,ref=docker.io/%s/python-cache:envd-%s",
-			viper.GetString(flag.FlagDockerOrganization),
-			version.GetVersionForImageTag())
-	}
-	return &res, nil
+	// We don't have remote cache for v1
+	return nil, nil
```

Comment:
LGTM, but should we also keep cache for v1?

Reply:
V1 changes frequently. The remote cache may not work as expected. I think we only need to maintain the remote cache for the stable version.

Reply:
SGTM